estate/models/house.py

```diff
@@ -0,0 +1,39 @@
+from odoo import models, fields
+
+class house(models.Model):
```

Comment:
follow PascalCase for python class names
```suggestion
class House(models.Model):
```